bot.py

```diff
@@ -645,19 +636,11 @@ async def clear_list_of_summoners(ctx):
     """Clear out summoners from the list"""
 
     try:
-        # for importing data from json file
-        file_data = ""
-        # initializing server id to a variable
         server_id = str(ctx.guild.id)
-
-        if os.path.getsize(json_path) > 0:
-            with open(json_path, "r") as file:
-                file_data = json.load(file)
-
-        if server_id in file_data.keys():
-            file_data[server_id].clear()
-            with open(json_path, "w") as file:
-                json.dump(file_data, file, indent=4)
+        member_list_query_result = (
+            session.query(TeamMembers).filter(TeamMembers.channel_id == server_id).one()
+        )
+        member_list_query_result.delete(session)
 
         # display list of summoners
         await display_current_list_of_summoners(ctx)
```

Comment:
@HelloFive Why are we displaying current list when there will be no list? 
don't copy paste